src/glue.c

```diff
@@ -991,7 +991,7 @@ void FuncDeclaration_toObjFile(FuncDeclaration *fd, bool multiobj)
     IRState irs(m, fd);
     Dsymbols deferToObj;                   // write these to OBJ file later
     irs.deferToObj = &deferToObj;
-    AA *labels = NULL;
+    void *labels = NULL;
```

Comment:
The * is on the "wrong" side of the whitespace in D (in C++ it was arguably in the right position).Perhaps as refactorings go, the stars might slowly migrate to the right.